features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumConfiguration.kt

```diff
@@ -335,6 +335,16 @@ data class RumConfiguration internal constructor(
             return this
         }
 
+        /**
+         * Enable/Disable the collection of graphQL payloads.
+         *
+         * By default the SDK will not collect these payloads.
+         */
+        fun captureGraphQLPayloads(enabled: Boolean): Builder {
```

Comment:
I don't think it should be here, because it is a technology-specific API. WDYT about moving it to `_RumInternalProxy` and calling only from `Apollo` module?

Also it can be just the attribute passed to the `startResource/stopResource`

Reply:
I've moved this to be a constructor parameter of the `DatadogApolloInterceptor`